src/main/kotlin/birdflop-presets.kt

```diff
@@ -0,0 +1,50 @@
+// This file adds the rgb.birdflop.com presets
+
+package chattore
+
+val birdflopGradients = mapOf(
+    "birdflop" to arrayOf(
+        "#084CFB",
+        "#ADF3FD"
+    ),
+    "SimplyMC" to arrrayOF (
```

Comment:
I just realized this is calling `arrayOF` but it should be `arrayOf`. All subsequent calls in this file are also `arrayOF`.